### PR TITLE
Introduce a danger file to check for `fit` and `fdescribe` in tests

### PR DESCRIPTION
**Pull request by orta**
I don't want to go overboard and try and introduce a big `Dangerfile` - so just trying to cover something I've accidentally shipped already in #5163

### CODE REVIEW

**Comment by orta** [comment]
Details re: credentials are in the CocoaPods Admin dmg

File: Dangerfile [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Don't let testing shortcuts get into master by accident
+
+fail("fdescribe left in tests") if `grep -r fdescribe spec/`.length > 1
```
Comment (segiddins) [inline Dangerfile]: use single quotes, and make sure RuboCop is checking the Dangerfile

Comment (segiddins) [inline Dangerfile]: also, I'd prefer to never use backticks in our code because shells are evil

Comment (orta) [inline Dangerfile]: got options on better ways to pull off this grep? ( not being cynical, would be interested on ways to do it better for Danger in it's entirety )

File: .travis.yml [hunk 2]
```diff
@@ -26,7 +26,9 @@ after_install:
 before_script:
   - bundle exec pod repo update --silent
 
-script: bundle exec rake
+script:
+  bundle exec rake
```
Comment (segiddins) [inline .travis.yml]: needs to be `-` so its a list instead of a concatenated string

**Comment by segiddins** [comment]
shouldn't Danger have failed on this?

**Comment by orta** [comment]
Yep!

**Comment by orta** [comment]
The CI is a bit weird here

**Comment by CocoaPodsBarista** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-kind="Error">
        
            3 Errors
        
      </th>
     </tr>
  </thead>
  <tbody>
        <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="true">fdescribe left in tests</td>
    </tr>
        <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="true">fit left in tests</td>
    </tr>
        <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="true">fdescribe left in tests</td>
    </tr>
    
    
  </tbody>
</table>
  

  

  




<p align="right" data-meta="generated_by_danger" data-base-commit="" data-head-commit="" >
  Generated by :no_entry_sign: <a href="https://github.com/danger/danger/">danger</a>
</p>

**Comment by orta** [comment]
Hah - just realised, Danger is failing this because she's also looking at the Dangerfile

**Comment by segiddins** [comment]
Made a few small changes & rebased, this is now good to merge from me -- @orta feel free to do so once CI has run

**Comment by orta** [comment]
Looks :+1:

**Pull request merged**